### ISSUE TITLE
Request a new payment method

### ISSUE THREAD

**Issue by Cakamaka111111** [remjx/coindrop#959]
Payment method name: 
Tether ( USDT ( ERC20 ) )
Payment method website: 
https://tether.to/en/
Link to icon in .SVG format (if available): 
https://seeklogo.com/images/T/tether-usdt-logo-FA55C7F397-seeklogo.com.png

Please i want to Request a new payment method for this gmail (  db.qp.income.qp.db@gmail.com  ) THANKS....

**Comment by TecnikOfficial** [comment]
i dont think this project is active.... i loved the concept wish it ws supported
